schemas/speciality.py

```diff
@@ -6,4 +6,5 @@
 class SpecialityListOut(BaseModel):
     university_id: int
     speciality_id: int
-    speciality_info: Dict[str, Union[int, str]]
```

Comment:
please make sure that order of sqlachemy columna are the same that in sql view
for ex: in view: faculty_id col - 1 st place, but there is this column on 3rd place